tests/test_core.py

```diff
@@ -299,26 +391,62 @@ def _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file,
                 scaper.trim(orig_wav_file.name, orig_jam_file.name,
                             orig_wav_file.name, orig_jam_file.name,
                             np.random.uniform(0, 1), np.random.uniform(3, 4))
+
                 scaper.generate_from_jams(orig_jam_file.name, gen_wav_file.name,
                                           save_isolated_events=True, 
                                           isolated_events_path=gen_events_path)
 
                 _validate_soundscape_and_event_audio(orig_wav_file, gen_wav_file, 
                     gen_events_path, orig_events_path)
 
+                # TODO
```

Comment:
See other 2 TODOs. Just want to document it somewhere what needs to go here eventually, if anything.

Reply:
Look at the following line, it's commented out but once supported should be uncommented